fdbclient/ManagementAPI.actor.cpp

```diff
@@ -517,6 +517,34 @@ bool isCompleteConfiguration(std::map<std::string, std::string> const& options)
 	       options.count(p + "storage_engine") == 1;
 }
 
+ACTOR Future<Void> disableBackupWorker(Database cx) {
```

Comment:
Is there an fdbcli status for backup workers? Can I ask if enabled/disabled?

Reply:
Or, its not needed because backup workers are enabled/disabled by the backup agent?